docs/api.md

```diff
@@ -19,6 +19,9 @@ Commands:
 |language|string|The currently supported languages are 'JavaScript', 'C#', and 'Java'.|
```

Comment:
Mixed feelings about the globalness here. Will you have troubles injecting ui and trees into test cases?  This is less of an issue with JavaScript.  Certainly be a lot less of a pain, though. I think it's worth experimenting with.

Reply:
No problems injecting for testing. In some ways its actually easier this way